### DISCUSSION TITLE
Does supabase run requests securely?

### DISCUSSION THREAD

**Discussion by mrvillage** [supabase/supabase#1918] [Questions] [answered]
Hello! I just want to know if the JavaScript library for supabase will run it's requests securely through HTTPS or some form of encryption or if they are plain requests over the web. Thanks! :)

**Comment by easy-street** [comment, marked as answer]
yes, the JavaScript library communicates with the hosted backend securely. all requests will be sent to the URL provided as the [`supabaseUrl` parameter](https://supabase.io/docs/reference/javascript/initializing#parameters) when [initializing the client](https://supabase.io/docs/reference/javascript/initializing#createclient). in production, this value is an `https` URL:

> The unique Supabase URL which is supplied when you create a new project in your project dashboard.

**Reply by mrvillage** [reply]
Oh perfect! Thank you!
